doc/release/upcoming_changes/15844.new_feature.rst

```diff
@@ -0,0 +1,5 @@
+crackfortran has support for inline attributes in type definition
+-----------------------------------------------------------------
+``crackfortran`` parser is now understanding inline attributes in
+type definition. There is no need anymore to put them on an
+additional line.
```

Comment:
"inline attribute" has a different meaning in Fortran than used here. Suggest rephrasing the note as follows:
```
f2py supports reading access type attributes from derived type statements
-------------------------------------------------------------------------
As a result, one does not need to use `public` or `private` statements to specify derived type access properties.
```

Reply:
Changed for your suggestion.